examples/running-tests-in-chrome-using-bitbucket-pipelines-ci/README.md

```diff
@@ -0,0 +1,14 @@
+# Example: Running Tests in Chrome Using Bitbucket Pipelines CI
+
+*A set of sample files that help you learn how to run TestCafe tests in the cloud using Chrome (Headless) with Bitbucket Pipelines CI.*
```

Comment:
help you learn?

Reply:
*A set of sample files that explains how to use Bitbucket Pipelines CI in Chrome (Headless) to run TestCafe tests in the cloud.*